demo_plugin/modelgauge/tests/demo_04_using_annotation_test.py

```diff
@@ -54,13 +53,14 @@ def make_test_items(self, dependency_helper: DependencyHelper) -> List[TestItem]
             test_items.append(TestItem(prompts=[prompt]))
         return test_items
 
-    def get_annotators(self) -> Mapping[str, Annotator]:
-        # Request the `DemoYBadAnnotator` be run, and key its results with "badness" for later.
-        return {"badness": DemoYBadAnnotator("badness_annotator")}
+    @classmethod
+    def get_annotators(cls) -> List[str]:
+        # Request the `DemoYBadAnnotator` be run.
+        return ["demo_annotator"]
 
     def measure_quality(self, item: TestItemAnnotations) -> Dict[str, float]:
         # Retrieve the annotation we asked for in `get_annotators`
-        annotation = item.interactions[0].response.completions[0].get_annotation("badness", DemoYBadAnnotation)
+        annotation = item.interactions[0].response.completions[0].get_annotation("demo_annotator", DemoYBadAnnotation)
```

Comment:
Can the `demo_annotator` string be pulled out of the class's list of annotator uids?

Reply:
Sure!